### PR TITLE
Enhance documentation

### PR DESCRIPTION
**Pull request by Zlopez**
This PR is updating Anitya documentation and adds tooltips to editable fields.

This PR also solves https://github.com/release-monitoring/anitya/issues/587

### CODE REVIEW

File: anitya/templates/project_new.html [hunk 1]
```diff
@@ -118,8 +118,29 @@ <h1>{{ context }} project</h1>
...
+    $("#homepage").prop('title', "Project homepage");
+    $("#version_url").prop('title', "URL where to look for new releases");
+    $("#version_prefix").prop('title', "Prefix filled here will be removed " +
+        "from retrieved version. For example version prefix " +
+        "'v' will convert 'v1.2.3' to '1.2.3'.");
+    $("#regex").prop('title', "Python regex to use for version retrieval. " + 
+        "Matched groups will be concatenated by '.'. " + 
+        "For example regex '(\d)_(\d)_(\d)' applied to '1_2_3' " +
+        "will convert version to '1.2.3'.");
+    $("#insecure").prop('title', "This should be set only when Version URL is HTTP");
```
Comment (jeremycline) [inline anitya/templates/project_new.html]: This should be "This should only be set when the version URL is HTTPS and the host does not have a trusted certificate"

**Comment by Zlopez** [comment]
When this PR is still open I added a new feature to documentation. The database schema is now generated using [sphinx-sadisplay](https://github.com/sphinx-contrib/sadisplay) extension.

**Comment by jeremycline** [comment]
Generally it's best to have a separate PR for additional features. I built the docs locally and it doesn't look like the image in rendering inline (it's a link).

We used a different library for Bodhi, so ideally we can use the same one for both (this or the existing one which is packaged for Fedora)

**Comment by Zlopez** [comment]
It is part of the documentation enhancement, so I think this is good place where to add it.

I will look at the Bodhi to see, how is this generated there.

**Comment by Zlopez** [comment]
I changed the database schema to be generated same as in Bodhi. It looks better now.

**Comment by Zlopez** [comment]
Weird, I tried to run the travis-ci locally using docker image - `travisci/ci-garnet:packer-1512502276-986baf0` and everything works, I don't know why the automatic build is failing.

**Comment by jeremycline** [comment]
> It is part of the documentation enhancement, so I think this is good place where to add it.

A PR should be a narrow change set. Typically, they're one commit unless there's a set of related changes that all need to land at once. Once one has been submitted and initially reviewed, it's best not to add to it. When I come back for a second review, my expectation is the the only changes will be in response to my comments. When new things show up along with those changes, it makes it difficult to review.

**Comment by Zlopez** [comment]
Sorry for this. I could remove the part with generation from this PR and move it to another PR, if this is more acceptable.

**Comment by Zlopez** [comment]
Removed the last commit. I will create a new PR from it.

**Pull request merged**